Slim/Networking/SimpleAsyncWS.pm

```diff
@@ -330,7 +330,7 @@ sub _continueListen {
 sub send {
 	my ($self, $buf) = @_;
 
-	main::INFOLOG && $log->is_info && $log->info("Sending on web socket : " . Dumper($buf));
```

Comment:
You could use `Data::Dump::dump()` instead, if you wanted to get a dump of that value.

Reply:
I've removed that now, I don't think that's necessery.